screens/account/VerifyEmail.tsx

```diff
@@ -1,23 +1,26 @@
-import { Button, Center, Heading, Text, VStack } from 'native-base';
-import { useEffect } from 'react';
-import { useRecoilState } from 'recoil';
+import { Button, Center, Heading, Input, Text, VStack } from 'native-base';
+import { useEffect, useState } from 'react';
+import { useSetRecoilState } from 'recoil';
 import { userPermissionLevelAtom } from '../../utils/atoms';
-import { sendAuthEmail, startWaitForVerificationPoll } from '../../xplat/api';
 import { auth } from '../../xplat/Firebase';
-import { UserStatus } from '../../xplat/types';
+import { confirmEmailCode, sendEmailCode } from '../../xplat/api';
 
 const VerifyEmail = () => {
-  const [userPermissionLevel, setUserPermissionLevel] = useRecoilState(
-    userPermissionLevelAtom
-  );
+  const setUserPermissionLevel = useSetRecoilState(userPermissionLevelAtom);
+  const [targCode, setTargCode] = useState<number>();
+  const [attemptCode, setAttemptCode] = useState<string>('');
+
+  useEffect(() => {
+    if (!targCode) sendEmailCode().then(setTargCode);
```

Comment:
```suggestion
    if (targCode !== undefined) sendEmailCode().then(setTargCode);
```
Even though `!targCode` is correct, industry standard for `undefined` checks is *strictly* equal to `undefined`.